gitlab_scripts/test_python.sh

```diff
@@ -21,8 +21,13 @@ date
 cd $BUILD_TYPE/src/unity/python
 make python_source
 cd ../../../..
-cp -a $BUILD_TYPE/src/unity/python/turicreate/test deps/env/lib/$2/site-packages/turicreate/
-cd deps/env/lib/$2/site-packages/turicreate/test
+
+PYTHON="deps/env/bin/python"
+PYTHON_MAJOR_VERSION=$(${PYTHON} -c 'import sys; print(sys.version_info.major)')
+PYTHON_MINOR_VERSION=$(${PYTHON} -c 'import sys; print(sys.version_info.minor)')
+PYTHON_VERSION="python${PYTHON_MAJOR_VERSION}.${PYTHON_MINOR_VERSION}"
+cp -a $BUILD_TYPE/src/unity/python/turicreate/test deps/env/lib/${PYTHON_VERSION}/site-packages/turicreate/
```

Comment:
Why is this copy necessary for tests? If `turicreate` has been successfully installed, why can't we just run the tests like `pytest --pyargs turicreate.test`? On that note, wouldn't it be safer to run `${PYTHON} -m pytest` as well?

Reply:
It's necessary because of the way we do testing, and the way we do packaging:

* Our wheels (when packaged) don't include tests, for size/brevity.
* Our tests expect to run in-package and do relative/local imports, which fail if the tests are outside the package.

It probably would be simpler to just include the tests in the packaged wheel. I doubt it saves much in terms of size.

It is probably better to run `${PYTHON} -m pytest`. I'll make that change, thanks.

Reply:
I opened #807 to track including tests in the wheel, which would allow us to simplify a lot of this stuff. However, tests are 2 MB including needed data, which does seem like a lot to add to every wheel installation.